corehq/apps/dhis2/models.py

```diff
@@ -0,0 +1,20 @@
+from dimagi.ext.couchdbkit import Document, StringProperty
+from dimagi.utils.couch.cache import cache_core
+
+
+class Dhis2Connection(Document):
```

Comment:
It would be useful if you could map out the data model for this feature so that we can get a better picture of how the pieces fit together. I think it would also be useful to validate that before diving into the code.